.codecov.yaml

```diff
@@ -0,0 +1,20 @@
+coverage:
+  # Commit status https://docs.codecov.io/docs/commit-status are used
+  # to block PR based on coverage threshold.
+  status:
+    project:
+      default:
+        target: 80
+        threshold: 1%
+    patch:
+      # Disable the coverage threshold of the patch, so that PRs are
+      # only failing because of overall project coverage threshold.
+      # See https://docs.codecov.io/docs/commit-status#disabling-a-status.
+      default: false
```

Comment:
Just curious why we want this?

Reply:
As in, seems like new code coming in should have coverage. The way I read it now is that our current is 96.8 (or something like that), having this would mean that we could keep adding code with no tests until coverage drops to 80?

Reply:
I copied this file from Serving.  Basically without this we saw codecov failing the `codecov/patch` check whenever any newly added line was uncovered.

Reply:
And isn't that what we want? :)

Reply:
discussed offline.